flink-runtime-web/web-dashboard/src/app/services/job.service.ts

```diff
@@ -168,7 +168,7 @@ export class JobService {
   private convertJob(job: JobDetail): JobDetailCorrect {
     const links: VerticesLink[] = [];
     let nodes: NodesItemCorrect[] = [];
-    if (job.plan?.nodes.length) {
+    if (job.plan?.nodes != null && job.plan.nodes.length) {
```

Comment:
```suggestion
    if (job.plan?.nodes?.length) {
```
This should work

Reply:
Indeed it works. I must've had cached scripts when testing it for the first time :( I will change it to what you're suggesting.